boards/arm/pan1783_pan1783a_pan1783apa_evb/doc/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _pan1783_evb:
+
+PAN1783, PAN1783A and PAN1783A-PA Evaluation Boards
+###################################################
+
+Overview
+********
+
+The PAN1783, PAN1783A and PAN1783A-PA Evaluation Boards (pan1783_evb,
+pan1783a_evb, pan1783a_pa_evb) are development tools for the PAN1783,
+PAN1783A and PAN1783A-PA Modules which are based on the nRF5340 chipset
+from Nordic Semiconductor.
+
+More information about the PAN1783, PAN1783A, PAN1783A-PA Modules and
+Evaluation Boards can be found on the `product website PAN1783`_ / `product website PAN1783A`_ / `product website PAN1783A-PA`_.
+
+.. figure:: img/pan1783_evb.jpg
+     :align: center
+     :alt: PAN1783 EVB
+
+     PAN1783 EVB (Credit: Panasonic)
+
+.. figure:: img/pan1783a_evb.jpg
+     :align: center
+     :alt: PAN1783A EVB
+
+     PAN1783A EVB (Credit: Panasonic)
+
+.. figure:: img/pan1783a_pa_evb.jpg
+     :align: center
+     :alt: PAN1783A-PA EVB
+
+     PAN1783A-PA EVB (Credit: Panasonic)
+
+Usage
+*****
+
+For detailed information, you can find the
+`pan1783_evb user guide`_ / `pan1783a_evb user guide`_ / `pan1783a_pa_evb user guide`_
+for the Evaluation Boards in the `Panasonic Wireless Connectivity Development Hub`_.
+
+The User Guide contains (amongst other things) detailed information about
+
+* pin mapping
+* powering options
+* breakout pin header interface
+* current consumption measurement
+* software development
+
+The schematics for the PAN1783/PAN1783A/PAN1783A-PA Evaluation Boards are
+available in the `download section PAN1783`_ / `download section PAN1783A`_ / `download section PAN1783A-PA`_
+of the `Panasonic Wireless Connectivity Development Hub`_.
+
+Programming and Debugging
+*************************
+
+Please use the ``pan1783_evb_cpuapp``, ``pan1783a_evb_cpuapp`` or
+``pan1783a_pa_evb_cpuapp`` for application core and ``pan1783_evb_cpunet``,
+``pan1783a_evb_cpunet`` or ``pan1783a_pa_evb_cpunet`` board configuration
+for network core when :ref:`build_an_application` and :ref:`application_run`.
+
+.. target-notes::
+.. _product website PAN1783: https://industry.panasonic.eu/products/devices/wireless-connectivity/bluetooth-low-energy-modules/pan1783-nrf5340
+.. _product website PAN1783A: https://industry.panasonic.eu/products/devices/wireless-connectivity/bluetooth-low-energy-modules/pan1783a-nrf5340
+.. _product website PAN1783A-PA: https://industry.panasonic.eu/products/devices/wireless-connectivity/bluetooth-low-energy-modules/pan1783a_pa-nrf5340
```

Comment:
these are HTTP 404 for me, can you check?

Reply:
Yes, you are right. Have to wait for the team to finish the website in background. Have removed specific links and added one that leads to portfolio overview.